### DISCUSSION TITLE
后宅设置了收集爱心和家具币，但是在游戏里执行却没有收集，只是点进去喂了食就退出了

### DISCUSSION THREAD

**Discussion by xfymw** [LmeSzinc/AzurLaneAutoScript#3480] [Q&A]
后宅方面家具也全部清空了
![1](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/147960045/289adf53-3f86-4438-beff-5143a6886fc2)

**Comment by guoh064** [comment]
#3506

**Reply by xfymw** [reply]
没明白你什么意思

**Reply by guoh064** [reply]
#3484
